packages/react-magma-dom/src/components/Datagrid/Datagrid.tsx

```diff
@@ -51,6 +53,7 @@ export interface DatagridRow {
    * Used to allow each unique column field as a key with the row content as the value
    */
   [key: string]: any;
+  tableTitle?: any;
```

Comment:
I think we want users to pass a node (React.ReactNode), and maybe also a string? but I don't think we should make this as broad as `any`

Reply:
We also need to add a description to explain to adopters when to use this